src/Core/Infrastructure/RealTime/Api/Hypermedia/HostHypermediaProvider.php

```diff
@@ -101,6 +101,22 @@ public function createForEventLog(mixed $response): ?string
      */
     public function createForTimelineEndpoint(mixed $response): string
     {
-        return $this->router->generate(static::ENDPOINT_HOST_TIMELINE, ['hostId' => $response->id]);
+        return $this->router->generate(self::ENDPOINT_HOST_TIMELINE, ['hostId' => $response->id]);
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function createForPerformanceDataEndpoint(mixed $data): string
+    {
+        return '';
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function createForStatusGraphEndpoint(mixed $data): string
+    {
+        return '';
```

Comment:
what is the purpose of those ? It will be implemetend later ?

Reply:
It was done regarding the interface

Reply:
Create abstract class and return null instead of empty string